packages/components/src/query-controls/index.tsx

```diff
@@ -59,6 +87,10 @@ export default function QueryControls( {
 					},
 				] }
 				onChange={ ( value ) => {
+					if ( typeof value !== 'string' ) {
+						return;
```

Comment:
Without this, there's a TS error:

<img width="1007" alt="Screen Shot 2022-12-21 at 6 01 29 PM" src="https://user-images.githubusercontent.com/4063887/209026397-c23e78f5-0b6c-429a-a9e2-d53abde760d0.png">